frame/contracts/src/lib.rs

```diff
@@ -427,8 +430,8 @@ pub trait Trait: frame_system::Trait {
 	/// The maximum size of a storage value in bytes.
 	type MaxValueSize: Get<u32>;
 
-	/// The maximum amount of gas that could be expended per block.
-	type BlockGasLimit: Get<Gas>;
+	/// Convert a weight value into a deductible fee based on the currency type.
```

Comment:
I think as the current implementation only support linear WeightToFee we should require this in the doc